<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listLiveBulk">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QListWidget" name="listCrumBulk">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>161</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="labelCry">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Crystals
Bulk</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDisto">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>50</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Distolate
Bulk</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCrum">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Crumble/Goo
Bulk</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listCryBulk">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>100</y>
      <width>161</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QListWidget" name="listDistoBulk">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>100</y>
      <width>161</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="labeLive">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Live Resin
Bulk</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCryUnit">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>270</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Crystals
Units</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDistoUnit">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>270</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Distolate
Units</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCrumUnit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Crumble/Goo
Units</string>
    </property>
   </widget>
   <widget class="QLabel" name="labeLiveUnit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>260</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Live Resin
Units</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listLiveUnit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>330</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QListWidget" name="listCrumUnit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>161</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QListWidget" name="listDistoUnit">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>330</y>
      <width>161</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QListWidget" name="listCryUnit">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>330</y>
      <width>161</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>968</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSend_To">
    <property name="title">
     <string>Send To</string>
    </property>
    <addaction name="actionDistolate"/>
    <addaction name="actionPackaging"/>
    <addaction name="actionPOS"/>
   </widget>
   <widget class="QMenu" name="menuAdd">
    <property name="title">
     <string>Add/Remove</string>
    </property>
    <addaction name="actionAdd_New_Product"/>
    <addaction name="actionRemove_Product"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSend_To"/>
   <addaction name="menuAdd"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionDistolate">
   <property name="text">
    <string>Distolate</string>
   </property>
  </action>
  <action name="actionPackaging">
   <property name="text">
    <string>Packaging</string>
   </property>
  </action>
  <action name="actionBulk">
   <property name="text">
    <string>Bulk</string>
   </property>
  </action>
  <action name="actionUnits">
   <property name="text">
    <string>Units</string>
   </property>
  </action>
  <action name="actionPOS">
   <property name="text">
    <string>POS</string>
   </property>
  </action>
  <action name="actionAdd_New_Product">
   <property name="text">
    <string>Add New Product</string>
   </property>
  </action>
  <action name="actionRemove_Product">
   <property name="text">
    <string>Remove Selected</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
